android-components/components/service/contile/src/test/java/mozilla/components/service/contile/ContileTopSitesProviderTest.kt

```diff
@@ -44,7 +44,7 @@ class ContileTopSitesProviderTest {
     fun `GIVEN a successful status response WHEN top sites are fetched THEN response should contain top sites`() =
         runTest {
             val client = prepareClient()
-            val provider = ContileTopSitesProvider(testContext, client)
+            val provider = ContileTopSitesProvider(testContext, false, client)
```

Comment:
We could avoid adding the `false` here once we order the list of parameters so that non-optional parameters appear before optional ones.